src/Indigo/Contracts/HomebaseLite/Impl/Vote.hs

```diff
@@ -57,25 +57,26 @@ verifyMinBalance :: HomebaseLiteEntrypoint [BalanceResponseItem]
 verifyMinBalance bris = do
   description [itu|
     Callback for the `balance_of` CPS view of FA2,
-    used to check the balance of a `participant` that used the `propose` entrypoint.
-    Takes as input the list of tuples containing the `owner` , `token_id` and `balance` .
+    used to check the governance token balance of a `participant` that used the
+    `propose` entrypoint.
+    Takes the list of tuples containing the `owner` , `token_id` and `balance` as input.
```

Comment:
```suggestion
    used to check the governance token's balance of a `participant` that used the
    `propose` entrypoint.
```
I think we can remove what the input is here, given that it is enforced by the FA2 spec.

Or perhaps we could say that instead:
```suggestion
    used to check the governance token balance of a `participant` that used the
    `propose` entrypoint. 
    Takes the response to the FA2 `balance_of` entrypoint as input.
```

Reply:
Let's remove it, it's documented anyway.